bigtable/tests/unit/test_table.py

```diff
@@ -16,6 +16,53 @@
 import unittest
 
 
+class Test___mutate_rows_request(unittest.TestCase):
```

Comment:
This test module should also have a test for the newly added `Table.mutate_rows`.